publish-game/index.md

```diff
@@ -0,0 +1,25 @@
+## How do I publish my game?
+
+When you are ready to publish your game, head over to your [dashboard](http://gamejolt.com/dashboard/) and select the game from the list under "Your Games". You will be taken to the game's management page. Here, you can publish the game by clicking/tapping "Unhide your game."
```

Comment:
should we also add in here that they'll need to unhide their distribution as well? if not, then this is only talking about the game page vs the game... so should we update it to say "game page" instead?